### PR TITLE
Mainnet configs - cheq denoms unifications

### PR DESCRIPTION
**Pull request by Eengineer1**

### CODE REVIEW

File: src/configs/chain_config.mainnet.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
   "votingPowerTokenUnit": "ncheq",
   "tokenUnits": {
     "ncheq": {
-      "display": "CHEQ",
+      "display": "cheq",
```
Comment (ankurdotb) [inline src/configs/chain_config.mainnet.json:21]: For display, we always stylise it as CHEQ (all caps) like ATOM...so not sure if this is correct

**Review by ankurdotb — CHANGES_REQUESTED** [review]
Likely reject?

File: src/configs/chain_config.mainnet.json (hunk 1 above)
Comment (Eengineer1) [reply src/configs/chain_config.mainnet.json:21]: In BDv2, display is the exact literal used by CoinGecko, so this is the correct one as fixed in [DEV-1090].
By default, BDv2 formats that in uppercase.

**Comment by Eengineer1** [comment]
> Likely reject?

Resolved.

**Comment by ankurdotb** [comment]
> > Likely reject?
> 
> Resolved.

Wait, should this not be CHEQ in all caps?

**Review by ankurdotb — APPROVED** [review]

**Comment by ankurdotb** [comment]
> > > > Likely reject?
> > > 
> > > 
> > > Resolved.
> > 
> > 
> > Wait, should this not be CHEQ in all caps?
> 
> No, not needed. It is formatted later, so it needs to match CoinGecko's to display correctly, which is in lowercase.

Thanks for that clarification, good to know. Approved!

**Pull request merged**